google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreOptions.java

```diff
@@ -63,17 +69,31 @@ public ServiceRpc create(DatastoreOptions options) {
     }
   }
 
+  @Nonnull
+  com.google.cloud.datastore.telemetry.TraceUtil getTraceUtil() {
+    return traceUtil;
+  }
+
+  @BetaApi
+  @Nonnull
+  public DatastoreOpenTelemetryOptions getOpenTelemetryOptions() {
+    return openTelemetryOptions;
+  }
+
   public static class Builder extends ServiceOptions.Builder<Datastore, DatastoreOptions, Builder> {
 
     private String namespace;
     private String databaseId;
 
+    @Nullable private DatastoreOpenTelemetryOptions openTelemetryOptions = null;
```

Comment:
would this really be nullable if we're setting it later on?

Reply:
This could be null i.e. if the user never calls Builder.setOpenTelemetryOptions(). It's the DatastoreOptions.openTelemetryOptions that is @NonNull.